File: src/main/java/alfheim/common/integration/travellersgear/TravellersGearAlfheimConfig.java
package alfheim.common.integration.travellersgear;

import alfheim.common.core.registry.AlfheimItems;
import cpw.mods.fml.common.event.FMLInterModComms;
import net.minecraft.item.ItemStack;
import vazkii.botania.common.item.ModItems;

public class TravellersGearAlfheimConfig {

	public static void loadConfig() {
		FMLInterModComms.sendMessage("TravellersGear", "registerTravellersGear_0", new ItemStack(ModItems.holyCloak));
		FMLInterModComms.sendMessage("TravellersGear", "registerTravellersGear_0", new ItemStack(ModItems.unholyCloak));
		FMLInterModComms.sendMessage("TravellersGear", "registerTravellersGear_0", new ItemStack(AlfheimItems.balanceCloak));
	}
}

File: src/main/java/alfheim/common/integration/travellersgear/TravellersGearAlfheimConfig.kt
package alfheim.common.integration.travellersgear

import alfheim.common.item.AlfheimItems
import cpw.mods.fml.common.event.FMLInterModComms
import net.minecraft.item.ItemStack
import vazkii.botania.common.item.ModItems

object TravellersGearAlfheimConfig {
	
	fun loadConfig() {
		FMLInterModComms.sendMessage("TravellersGear", "registerTravellersGear_0", ItemStack(ModItems.holyCloak))
		FMLInterModComms.sendMessage("TravellersGear", "registerTravellersGear_0", ItemStack(ModItems.unholyCloak))
		FMLInterModComms.sendMessage("TravellersGear", "registerTravellersGear_0", ItemStack(AlfheimItems.balanceCloak))
	}
}